pachyderm/templates/pachd/storage-secret.yaml

```diff
@@ -1,44 +1,47 @@
 apiVersion: v1
 data:
   {{- if eq .Values.pachd.storage.backend "GOOGLE" }}
-  google-bucket: {{ required "Google bucket Required when storage backend GOOGLE" .Values.pachd.storage.google.googleBucket | toString | b64enc }}
-  google-cred: {{ .Values.pachd.storage.google.googleCred | toString | b64enc }}
-  {{- end -}}
-  {{- if eq .Values.pachd.storage.backend "MINIO" }}
-  minio-bucket: {{ .Values.pachd.storage.minio.minioBucket | toString | b64enc }}
-  minio-endpoint: {{ .Values.pachd.storage.minio.minioEndpoint | toString| b64enc }}
-  minio-id: {{ .Values.pachd.storage.minio.minioID | toString| b64enc }}
-  minio-secret: {{ .Values.pachd.storage.minio.minioSecret | toString | b64enc }}
-  minio-secure: {{ .Values.pachd.storage.minio.minioSecure | toString | b64enc }}
-  minio-signature: {{ .Values.pachd.storage.minio.minioSignature | toString | b64enc }}
-  {{- end -}}
-  {{- if eq .Values.pachd.storage.backend "AMAZON" }}
-  amazon-bucket: {{ .Values.pachd.storage.amazon.amazonBucket | toString | b64enc }}
-  amazon-distribution: {{ .Values.pachd.storage.amazon.amazonDistribution | toString | b64enc }}
-  amazon-id: {{ .Values.pachd.storage.amazon.amazonID | toString | b64enc }}
-  amazon-region: {{ .Values.pachd.storage.amazon.amazonRegion | toString | b64enc }}
-  amazon-secret: {{ .Values.pachd.storage.amazon.amazonSecret | toString | b64enc }}
-  amazon-token: {{ .Values.pachd.storage.amazon.amazonToken | toString | b64enc }}
+  google-bucket: {{ required "Google bucket Required when storage backend GOOGLE" .Values.pachd.storage.google.bucket | toString | b64enc | quote }}
```

Comment:
Why are we quoting some values and not others?

Reply:
I think that they should all be quoted.

Although I have not managed to get any of the encodings I have tested all the way through, it _is_ possible to have a valid Base64 encoding of bytes which is itself a non-string YAML literal (e.g. hex `f7df7d` encodes to`9999` and hex `b6bb9e` encodes to `true`). Seems safer to always quote than risk a type error.